lib/connection.h

```diff
@@ -141,8 +141,9 @@ class Connection : public QObject {
      *         it's an Invite rather than Join - to room pointers
      * \sa allRooms, rooms, roomsWithTag
      */
-    [[deprecated("Use allRooms(), roomsWithTag() or rooms(joinStates) instead")]]
-    QHash<QPair<QString, bool>, Room*> roomMap() const;
+    [[deprecated("Use allRooms(), roomsWithTag() or rooms(joinStates) "
+                 "instead")]] QHash<QPair<QString, bool>, Room*>
+    roomMap() const;
```

Comment:
Clang-format is very bad with `[[attributes]]`, which is why I would advise against automatic formatting. This signature, in particular, should certainly be left intact.